core/src/test/java/org/openapitools/openapidiff/core/AddPropPutDiffTest.java

```diff
@@ -1,21 +0,0 @@
-package org.openapitools.openapidiff.core;
-
-import static org.openapitools.openapidiff.core.TestUtils.assertOpenApiAreEquals;
-import static org.openapitools.openapidiff.core.TestUtils.assertOpenApiBackwardIncompatible;
-
-import org.junit.jupiter.api.Test;
-
-public class AddPropPutDiffTest {
-  private final String OPENAPI_DOC1 = "add-prop-put-1.yaml";
-  private final String OPENAPI_DOC2 = "add-prop-put-2.yaml";
-
-  @Test
-  public void testDiffSame() {
-    assertOpenApiAreEquals(OPENAPI_DOC1, OPENAPI_DOC1);
-  }
-
-  @Test
-  public void testDiffDifferent() {
```

Comment:
I don't think we should remove this test. In fact they should be kept to check that the two docs are indeed compatible.

So we should have a `testFieldAdditionalInPutApiIsCompatible` or something similar

Reply:
@mas-chen Good point about not losing a test. I think my #546 PR will address this.

Reply:
Fair enough. I've made your suggested changes. Look ok?

Reply:
Looks good to me but I'm not a contributor, would be good to get the help of one to merge this fix